src/app/(proper_react)/(redesign)/MobileShell.tsx

```diff
@@ -125,97 +126,119 @@ export const MobileShell = (props: Props) => {
           className={styles.mainMenuLayer}
           aria-label={l10n.getString("mobile-menu-label")}
         >
-          <div className={styles.mainMenu}>
-            {props.session ? (
-              <>
-                <ul>
-                  <li key="home">
-                    <PageLink
-                      href="/user/dashboard"
-                      activeClassName={styles.isActive}
-                      hasTelemetry={{ link_id: "navigation_dashboard" }}
-                    >
-                      {l10n.getString("main-nav-link-dashboard-label")}
-                    </PageLink>
-                  </li>
-                  {props.countryCode === "us" && (
-                    <li key="how-it-works">
+          {props.session &&
+          props.enabledFeatureFlags.includes("SidebarNavigationRedesign") ? (
+            <div className={styles.navbarListWrapper}>
+              <NavbarListAuthenticated
+                l10n={l10n}
+                countryCode={props.countryCode}
+                enabledFeatureFlags={props.enabledFeatureFlags}
+              />
+              <div className={styles.premiumCta}>
+                <UpsellBadge
+                  monthlySubscriptionUrl={props.monthlySubscriptionUrl}
+                  yearlySubscriptionUrl={props.yearlySubscriptionUrl}
+                  subscriptionBillingAmount={props.subscriptionBillingAmount}
+                  // The last scan date is too noisy on mobile, so don't show it there:
+                  lastScanDate={null}
+                />
+              </div>
+            </div>
+          ) : (
+            <div className={styles.mainMenu}>
```

Comment:
I'm assuming that there were no changes in this branch?

Reply:
Yes, that’s right.